packages/stable/src/__tests__/api/views.int.spec.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 Cognite AS
+
+import { ViewCreateDefinition } from '../../api/views/types.gen';
+import CogniteClient from '../../cogniteClient';
+import { setupLoggedInClient } from '../testUtils';
+
+describe('Views integration test', () => {
+  let client: CogniteClient;
+  const timestamp = Date.now();
+
+  const TEST_SPACE_NAME = `Views_integration_test_${timestamp}`;
+  const TEST_CONTAINER_NAME = `Views_integration_test_container${timestamp}`;
+
+  const viewCreationDefinition: ViewCreateDefinition = {
+    externalId: 'test_view',
```

Comment:
Should we append the timestamp here as well?

Reply:
I guess it doesn't matter as long as the space is unique

Reply:
i think its still good to do!